src/execution_plan/execution_plan_build/build_match_op_tree.c

```
@@ -78,40 +78,50 @@ static OpBase *_ExecutionPlan_ProcessQueryGraph
 			AlgebraicExpression *ae_src =
-				AlgebraicExpression_RemoveSource (&exps[0]) ;
-			ASSERT(AlgebraicExpression_DiagonalOperand (ae_src, 0)) ;
+				AlgebraicExpression_RemoveSource(&exps[0]);
+			ASSERT(AlgebraicExpression_DiagonalOperand(ae_src, 0));
 
-			const char *label = AlgebraicExpression_Label (ae_src) ;
-			const char *alias = AlgebraicExpression_Src (ae_src) ;
-			ASSERT (label != NULL) ;
-			ASSERT (alias != NULL) ;
+			const char *label = AlgebraicExpression_Label(ae_src);
+			const char *alias = AlgebraicExpression_Src(ae_src);
```

Cppcheck (2.20.0)

[error] Common realloc mistake

(memleakOnRealloc)